app/controllers/locations_controller.rb

```diff
@@ -20,4 +20,24 @@ def create
   def show
     @location = Location.find(params[:id])
   end
+
+  def edit
+    @location = Location.find(params[:id])
+  end
+
+  def update
+    location = Location.find(params[:id])
+    location.update({
+      city: params[:location][:city],
+      square_footage: params[:location][:square_footage],
+      lead_wall: params[:location][:lead_wall]
+      })
+      location.save
+      redirect_to "/locations/#{location.id}"
+  end
```

Comment:
Many of our methods look super similar. I'm curious if there's a way for us to DRY this out a bit.